opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -94,6 +94,32 @@ def shutdown(self) -> None:
             sp.shutdown()
 
 
+class SpanStatus(trace_api.SpanStatus):
+    """See `opentelemetry.trace.SpanStatus`.
+
+    Represents the status of a finished Span
+
+    Args:
+        canonical_code: Represents the canonical set of status codes of a finished Span, following the Standard GRPC codes
+        description: Description of this Status
+        is_ok: Use to determine if the span was an error or not
+    """
+
+    def __init__(self, canonical_code=0, description=None, is_ok=True):
```

Comment:
I'm confused, I thought `is_ok` is deduced from the code?

Reply:
I created https://github.com/open-telemetry/opentelemetry-specification/pull/297 in the spec, but I think the `is_ok` getter of the code should just use canonical_code.